byterun/sys.c

```diff
@@ -368,6 +368,10 @@ CAMLprim value caml_sys_get_argv(value unit)
 
 void caml_sys_init(char_os * exe_name, char_os **argv)
 {
+  /* Initialises the caml_win32_* globals on Windows with the version of
+     Windows which is running */
+  caml_probe_os_version();
+  caml_setup_terminal();
 #ifdef CAML_WITH_CPLUGINS
```

Comment:
Guarding this with `#ifdef _WIN32` would make @nojb and I happy, I think.

Reply:
This is now done - it makes me happy too!